cmd/nf-dump-pcap/main.go

```diff
@@ -15,14 +15,14 @@ import (
 
 	"github.com/google/gopacket"
```

Comment:
I don't use anything under the `cmd` directory (they're just wrappers for cli programs), and `gopacket` isn't used elsewhere. The main repository doesn't complain when building, and I didn't see any other transitive dependencies.